packages/ui-library/src/components/toggle-switch/index.ts

```diff
@@ -91,32 +91,32 @@ export class BlrToggleSwitch extends LitElement {
     }
   };
 
-  @state() protected active = false;
+  @state() protected checked = false;
 
   protected handlePress = () => {
     if (!this.disabled && !this.readonly) {
-      this.active = true;
+      this.checked = true;
       this.currentCheckedState = !this.currentCheckedState;
     }
   };
 
   protected handleRelease = () => {
     if (!this.disabled && !this.readonly) {
-      this.active = false;
+      this.checked = false;
```

Comment:
This has nothing to do with the checked property being renamed to active, right? Because it should be "active" now and not "checked".

Reply:
This is unrelated. The checked state is necessary here to display the the visual active state(black) of the component which was  missing before.